### ISSUE TITLE
Is there a way to get the current learning rate from an Estimator

### ISSUE THREAD

**Issue by Doragd** [tensorflow/tensorflow#56124]
TF version: tf 1.15

I have spent so much time to find the solution .... BUT FIND NOTHING...

A similar problem was raised from StackOverflow, but no one can answer it.... 
https://stackoverflow.com/questions/60244149/is-there-a-way-to-get-the-current-learning-rate-from-an-estimator

PLEASE HELP ME....

IT IS  REALLY UPSET FOR ME to use TensorFlow

**Comment by sushreebarsa** [comment]
@Doragd 
We see that you are using old version of tensorflow(1.15) which is not actively supported, we recommend that you upgrade to 2.4 or later version and let us know if the issue still persists in newer versions. Estimators are not recommended for new code. Please refer to this [link](https://www.tensorflow.org/guide/estimator) for more details. For any further queries related to TF v1.x issue please post the issue on [TF Forum](https://discuss.tensorflow.org/) where there is a larger community to get you the right help. Thank you!

**Comment by Doragd** [comment]
Thanks a lot anyway.
